datalabeling/create_annotation_spec_set_test.py

```diff
@@ -33,4 +34,11 @@ def test_create_annotation_spec_set(capsys):
     # Delete the created annotation spec set.
     annotation_spec_set_name = response.name
     client = datalabeling.DataLabelingServiceClient()
+
+    # If provided, use a provided test endpoint - this will prevent tests on
+    # this snippet from triggering any action by a real human
+    if 'DATALABELING_ENDPOINT' in os.environ:
+        opts = ClientOptions(api_endpoint=os.getenv('DATALABELING_ENDPOINT'))
+        client = datalabeling.DataLabelingServiceClient(client_options=opts)
```

Comment:
would it make sense to move the other client creation to an else so we aren't creating a client twice?

Reply:
Normally yes, but in this case we want the part that is shown on the docs to look normal (which is why we hide it in the region tags)